README.md

```diff
@@ -39,6 +39,28 @@ jobs:
         branch: ${{ github.ref }}
 ```
 
+An example workflow to use the branch parameter to push the changes to specified branch e.g. an PR related branch:
+
+```yaml
+jobs:
```

Comment:
Can we add the event that would likely be used to trigger this action as `head_ref` is only set for specific events.

```
The head_ref or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is either pull_request or pull_request_target.
```
See: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
Done